test/test_orderly_web_api.py

```diff
@@ -0,0 +1,46 @@
+from src.orderly_web_api import OrderlyWebAPI
+import pytest
+import requests
+
+
+def get_montagu_token():
```

Comment:
Getting a Montagu token manually here. This could use the Montagu Python client though. I guess we need to package both these clients up and send them to PyPI?

Reply:
Ultimately yes, though sending to PyPI is nice and easy :)

The way I did this in the montagu/orderly R client was:

* orderlyweb client accepts `token` as either a callback to create a token or a string: https://github.com/vimc/orderlyweb/blob/master/R/api_client.R#L80-L86
* montagu client gets a token from the montagu API https://github.com/vimc/montagu-r/blob/master/R/orderly.R#L25-L31

Reply:
I'll make tickets to PyPI-ify these repos.

Reply:
Sorry, did you want me to change this to optionally take a callback as well as string?